src/NuGet.Services.SearchService/Controllers/SearchController.cs

```diff
@@ -81,7 +84,8 @@ public async Task<HttpResponseMessage> GetStatusAsync(HttpRequestMessage request
             string q = null,
             string sortBy = null,
             bool? luceneQuery = true,
-            bool? debug = false)
+            bool? debug = false,
+            string packageType = null)
```

Comment:
Super nit: let's be consistent with the V3 route and keep `debug` last, and place `packageType` second to last.